### DISCUSSION TITLE
Add "Times Played" as a tracking metric on song select screen

### DISCUSSION THREAD

**Discussion by AriizuCA** [ppy/osu#22069] [Ideas] [answered]
I think it could be nice to see the number of times players have attempted a map, similar to the stats that are already kept for "most played beatmaps" on profile pages, on the song select screen.  Since the numbers are already saved, it would be nice to see them displayed somewhere without the need to hit the "show more" button on the website until you eventually find the map you were searching for.

**Comment by peppy** [comment, marked as answer]
We do plan on tracking play statistics to a more detailed level in the future. Should be implemented at a realm level separate to score storage (could probably be attached to the beatmap, but would require some structural consideration).
